package.json

```diff
@@ -7,11 +7,11 @@
   "scripts": {
     "build": "node-gyp rebuild",
     "clean": "node-gyp clean",
-    "test": "./node_modules/.bin/mocha -b -t 40000 --reporter spec"
+    "test": "mocha -b -t 40000 --reporter spec"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/ibmruntimes/zrexx.git"
+    "url": "git+https://github.com/ibmruntimes/zrexx.git"
```

Comment:
Was this change also made by `npm pkg fix`? The existing url already correctly follows the format outlined in the [npm docs](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#repository).